src/services/email-verification-service.ts

```diff
@@ -0,0 +1,53 @@
+// NB: use native axios to pre-empt OPTIONS requests
+// because Million Verifier doesn't support them
+import axios from 'axios'
+
+enum ResultCodes {
+  OK = 'ok',
+  CATCH_ALL = 'catch_all',
+  UNKNOWN = 'unknown',
+  ERROR = 'error',
+  DISPOSABLE = 'disposable',
+  INVALID = 'invalid'
+}
+/**
+ * Class that provides integration with the Million Verifier API.
+ * Ref: https://developer.millionverifier.com/
+ */
+export default class EmailVerificationService {
+  /**
+   * Verifies an email address in real time.
+   * @param email the email address to verify
+   * @param apiUrl the API URL for the Million Verifier API
+   * @param apiKey the API key for the Million Verifier API
+   * @param timeout the timeout for the Million Verifier API
+   * @returns whether the email address is valid
+   */
+  static async isValidEmail (
+    email: string,
+    apiUrl = 'https://api.millionverifier.com/api/v3',
```

Comment:
I don't think we need external config for this since there's only 1 endpoint (no dev or test).

To prevent burning up credits while testing, this method automatically returns "valid" if no API key is configured.